Sprint-2/implement/lookup.js

```diff
@@ -1,5 +1,23 @@
-function createLookup() {
-  // implementation here
+function createLookup(currencyCodes) {
+  const createdObject = {};
+  let createdArray = [];
+  for (let i = 0; i < currencyCodes.length; i++) {
+    createdArray = currencyCodes[i];
+    console.log(createdArray);
+    const splitArray = createdArray.map((element) => [element]); //splits an array to arrayes of its elements
+
+    const key = splitArray[0].join(); //converts an array to a string
+    const value = splitArray[1].join(); ////converts an array to a string
```

Comment:
What is `createdArray`?

Code in lines 6-10 can greatly simplified.

Reply:
Thanks CJ, I simplified my codes. Also it seems "createdArray" is useless, so I removed it.